module/VuFind/src/VuFind/Cover/Loader.php

```diff
@@ -254,6 +270,8 @@ protected function getImageSettingsFromLegacyArgs($args)
             'issn' => $args[6],
             'oclc' => $args[7],
             'upc' => $args[8],
+            'nbn' => $args[9],
+            'ismn' => $args[10],
```

Comment:
This should not be necessary -- since these were not arguments in the legacy code, you shouldn't need to translate them to the new format.

Reply:
I have removed these two lines.